### ISSUE TITLE
A non-numeric value encountered - Timber\Timber::render()

### ISSUE THREAD

**Issue by reinisg** [timber/timber#2008]
Hello,
I`m using Wordpress 5.2 with timber 1.9.4 (additional debug bar and timber debug bar plugins).
I have single-shop.php with:
`<?php
$context = Timber::get_context();
$timber_post = Timber::query_post();
$context['post'] = $timber_post;
Timber::render( 'pages/single-shop.twig', $context );`

It`s working fine, but if I opened debug tool, I have php errors:
- https://imgur.com/3XDv1Pz
- https://imgur.com/rFpvlRY

What I`m doing wrong, is there any solution?

Regards,
Reinis.

**Comment by jarednova** [comment]
@reinisg that is a great question. Can you disable debug bar and see if there are still PHP errors hitting the logs?

**Comment by reinisg** [comment]
It was clean.
But I did manage to resolve this issue.
I deactivated debug plugins, entered in page and saw image is replaced with 'A non-numeric value encountered'. In twig file was written `image.sizes.large-width`, I replaced this code with `image.sizes['large-width']` and image was back. Also I reactivated plugins and in debug bar php errors was gone.

Thank you for a reply and advice!
Regards,
Reinis.
